### PR TITLE
chore: upgrade deps and fix test

### PR DESCRIPTION
**Pull request by popomore**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+node_modules
+coverage
```
Comment (atian25) [inline .eslintignore:2]: run
logs

**Review by atian25 — APPROVED** [review]

File: .eslintignore (hunk 1 above)
Comment (popomore) [reply .eslintignore:2]: eslint 不需要吧？

Comment (atian25) [reply .eslintignore:2]: 噢噢，看错

**Comment by popomore** [comment]
https://github.com/eggjs/egg/issues/309

**Comment by popomore** [comment]
这个 appveyor 过了，没回调

**Pull request merged**